js_modules/dagster-ui/packages/ui-core/src/assets/AssetMaterializationUpstreamData.tsx

```diff
@@ -146,13 +147,15 @@ export const AssetMaterializationUpstreamData = ({
   assetKey: AssetKeyInput;
   timestamp?: string;
 }) => {
+  const skip = !timestamp;
   const result = useQuery<
     AssetMaterializationUpstreamQuery,
     AssetMaterializationUpstreamQueryVariables
   >(ASSET_MATERIALIZATION_UPSTREAM_QUERY, {
     variables: {assetKey: {path: assetKey.path}, timestamp},
-    skip: !timestamp,
+    skip,
   });
+  useBlockTraceOnQueryResult(result, 'AssetMaterializationUpstreamQuery', {skip});
```

Comment:
similar thoughts about this one - separate query specifically because it's very slow and doesn't matter a whole lot